src/cs/sqlite3_pinvoke.cs

```diff
@@ -1218,15 +1220,15 @@ private static class NativeMethods
 
         private static bool TryLoadFromDirectory(string dllName, string baseDirectory)
         {
-            System.Diagnostics.Debug.Assert(!string.IsNullOrWhiteSpace(dllName), "dllName is null or empty.");
```

Comment:
That API is not available in .NET 3.5 (the platform I am targeting). So you can take this change or leave it :)